File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/addCustomers/map/interfaces2/IPositiveNegativeListener.java
package com.tiff.tiffinbox.Seller.addCustomers.map.interfaces2;


@FunctionalInterface
public interface IPositiveNegativeListener {
    void onPositive();

   default void onNegative(){}


    public static final class DefaultImpls {
        public static void onNegative(IPositiveNegativeListener $this) {
        }
    }
}

File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/addCustomers/map/interfaces2/IPositiveNegativeListener.kt
package com.tiff.tiffinbox.Seller.addCustomers.map.interfaces2

fun interface IPositiveNegativeListener {
    fun onPositive()
    fun onNegative() {}

    object DefaultImpl {
        @JvmStatic
        fun onNegative(`$this`: IPositiveNegativeListener?) {}
    }
}